__TESTS__/unit/actions/Flag.test.ts

```diff
@@ -217,6 +219,103 @@ describe('Tests for Transformation Action -- Flag', () => {
     expect(url).toBe('http://res.cloudinary.com/demo/image/upload/fl_keep_iptc/sample');
   });
 
+  it('Creates a cloudinaryURL with custom flag', () => {
+    const url = new TransformableImage()
+      .setConfig(CONFIG_INSTANCE)
+      .addFlag(Flag.custom('myFlag'))
+      .setPublicID('sample')
+      .toURL();
+
+    expect(url).toBe('http://res.cloudinary.com/demo/image/upload/fl_myFlag/sample');
+  });
+
+  it('Creates a cloudinaryURL with flag layer', () => {
+    const url = new TransformableImage()
+      .setConfig(CONFIG_INSTANCE)
+      .addFlag(Flag.layerApply())
+      .setPublicID('sample')
+      .toURL();
+
+    expect(url).toBe('http://res.cloudinary.com/demo/image/upload/fl_layer_apply/sample');
+  });
+
+  it('Creates a cloudinaryURL with Flag hlsv3', () => {
+    const url = new TransformableVideo()
+      .setConfig(CONFIG_INSTANCE)
+      .setAssetType('video')
+      .addFlag(Flag.hlsv3())
+      .setPublicID('sample')
+      .toURL();
+
+    expect(url).toBe('http://res.cloudinary.com/demo/video/upload/fl_hlsv3/sample');
+  });
+
+  it('Creates a cloudinaryURL with Flag keepDar', () => {
+    const url = new TransformableVideo()
+      .setConfig(CONFIG_INSTANCE)
+      .setAssetType('video')
+      .addFlag(Flag.keepDar())
+      .setPublicID('sample')
+      .toURL();
+
+    expect(url).toBe('http://res.cloudinary.com/demo/video/upload/fl_keep_dar/sample');
+  });
+
+  it('Creates a cloudinaryURL with Flag noStream', () => {
+    const url = new TransformableVideo()
+      .setConfig(CONFIG_INSTANCE)
+      .setAssetType('video')
+      .addFlag(Flag.noStream())
+      .setPublicID('sample')
+      .toURL();
+
+    expect(url).toBe('http://res.cloudinary.com/demo/video/upload/fl_no_stream/sample');
+  });
+
+  it('Creates a cloudinaryURL with Flag mono', () => {
+    const url = new TransformableVideo()
+      .setConfig(CONFIG_INSTANCE)
+      .setAssetType('video')
+      .addFlag(Flag.mono())
+      .setPublicID('sample')
+      .toURL();
+
+    expect(url).toBe('http://res.cloudinary.com/demo/video/upload/fl_mono/sample');
+  });
+
+  it('Creates a cloudinaryURL with Flag splice', () => {
+    const url = new TransformableVideo()
+      .setConfig(CONFIG_INSTANCE)
+      .setAssetType('video')
+      .addFlag(Flag.splice())
+      .setPublicID('sample')
+      .toURL();
+
+    expect(url).toBe('http://res.cloudinary.com/demo/video/upload/fl_splice/sample');
+  });
+
+  it('Creates a cloudinaryURL with Flag waveform', () => {
+    const url = new TransformableVideo()
+      .setConfig(CONFIG_INSTANCE)
+      .setAssetType('video')
+      .addFlag(Flag.waveform())
+      .setPublicID('sample')
+      .toURL();
+
+    expect(url).toBe('http://res.cloudinary.com/demo/video/upload/fl_waveform/sample');
+  });
```

Comment:
Can we combine all these tests into one? They all do the very same, simple thing

even a single image with 10-20 addFlag() would shorten this file quite a bit.